tests/Controller/UsersControllerTest.php

```diff
@@ -149,6 +152,90 @@ public function testDeleteUserWithAdminAccessDeletesSuccessfully()
         $this->assertNull($usersController->deleteUser($request, $db));
     }
 
+    public function testDeleteTalkCommentsWithoutBeingLoggedInThrowsException(): void
+    {
+        $this->expectException(AuthenticationException::class);
+        $this->expectExceptionMessage('You must be logged in to perform this operation.');
+        $this->expectExceptionCode(Http::UNAUTHORIZED);
```

Comment:
What do you think about setting these expectations as late as possible?

For reference, see https://thephp.cc/dates/2019/11/symfonycon/phpunit-best-practices.

Reply:
Yep, sounds good to me. Might add another commit moving the expectations in the other cases then